main.py

```diff
@@ -1,15 +1,77 @@
-from typing import Union
-
 from fastapi import FastAPI
+from pyathena import connect
+from typing import Union
+import pandas as pd
+import base64
+import boto3
+import redis
 
 app = FastAPI()
+r = redis.Redis(host='localhost', port=6379, db=0)
+athena_client = boto3.client('athena')
+s3_client = boto3.client('s3')
+
+AWS_DATA_CATALOG = "AwsDataCatalog"
+AWS_SCHEMA_DATABASE_NAME = "ensembl-parquet-meta-schema"
+AWS_S3_OUTPUT_DIR = "s3://ensembl-athena-results/"
+
+@app.get("/api/v1")
+def root():
+    return "Ensembl's data lakehouse backend"
+
+
+@app.get("/api/v1/data")
+async def read_all_filters():
+    query_response = athena_client.list_table_metadata(CatalogName=AWS_DATA_CATALOG, DatabaseName=AWS_SCHEMA_DATABASE_NAME)["TableMetadataList"]
+    # remove unnecessary data from AWS response
+    for table in query_response:
+        del table["CreateTime"]
+        del table["LastAccessTime"]
+        del table["TableType"]
+        del table["Parameters"]
+    return query_response
+
+
+@app.get("/api/v1/{data}/columns")
```

Comment:
Same comment, and I would ass that {data} could be misleading with previous route, if it's a `data_type` then change the name for this. 
```suggestion
@app.get("/{data_type}/columns")
```